### PR TITLE
Add Spanner region tags.

### PR DESCRIPTION
**Pull request by jmdobry**
Related:

* ~~googleapis/nodejs-spanner#136~~ merged
* ~~GoogleCloudPlatform/ruby-docs-samples#244~~ merged
* ~~GoogleCloudPlatform/python-docs-samples#1376~~ merged
* ~~GoogleCloudPlatform/java-docs-samples#1045~~ merged
* GoogleCloudPlatform/dotnet-docs-samples#454
* ~~GoogleCloudPlatform/php-docs-samples#562~~ merged

### CODE REVIEW

File: spanner/spanner_quickstart/main.go [hunk 1]
```diff
@@ -3,6 +3,7 @@
 // license that can be found in the LICENSE file.
 
 // Sample spanner_quickstart is a basic program that uses Cloud Spanner.
+// [START spanner_quickstart]
```
Comment (broady) [inline spanner/spanner_quickstart/main.go]: Please move this above the doc comment, or below it. As it is, it'll be included in package docs.

File: spanner/spanner_snippets/snippet.go [hunk 2]
```diff
@@ -80,9 +81,11 @@ func createDatabase(ctx context.Context, w io.Writer, adminClient *database.Data
 	}
 	fmt.Fprintf(w, "Created database [%s]\n", db)
 	return nil
+	// [END spanner_create_database]
```
Comment (broady) [inline spanner/spanner_snippets/snippet.go]: Move the end tags above the final return statement.

**Comment by broady** [comment]
Please run gofmt over the CL after the comments are addressed.

Something like this, maybe, if you don't have Go installed:

```
docker run -v $PWD:/gosrc golang gofmt -w $(find /gosrc -name '*.go')
```

File: spanner/spanner_quickstart/main.go (hunk 1 above)
Comment (jmdobry) [reply spanner/spanner_quickstart/main.go]: Done

File: spanner/spanner_snippets/snippet.go (hunk 2 above)
Comment (jmdobry) [reply spanner/spanner_snippets/snippet.go]: Done

**Comment by jmdobry** [comment]
Done

**Pull request merged**